client/blocks/login/login-form.jsx

```diff
@@ -857,207 +937,225 @@ export class LoginForm extends Component {
+										autoComplete="current-password"
+										className={ clsx( {
+											'is-error': requestError && requestError.field === 'password',
+										} ) }
+										onChange={ this.onChangeField }
+										id="password"
+										name="password"
+										ref={ this.savePasswordRef }
+										value={ this.state.password }
+										disabled={ isFormDisabled }
+										tabIndex={ isPasswordHidden ? -1 : undefined /* not tabbable when hidden */ }
+									/>
+
+									{ requestError && requestError.field === 'password' && (
+										<FormInputValidation isError text={ this.renderPasswordValidationError() } />
+									) }
+								</div>
+							</div>
 
-					{ ! isBlazePro && <p className="login__form-terms">{ socialToS }</p> }
-					{ isWoo && ! isPartnerSignup && ! isWooPasswordless && this.renderLostPasswordLink() }
-					<div className="login__form-action">
-						<FormsButton
-							primary
-							busy={ ! isWoo && isSendingEmail }
-							disabled={ isSubmitButtonDisabled }
-						>
-							{ isWoo && isSendingEmail ? <Spinner /> : this.getLoginButtonText() }
-						</FormsButton>
-					</div>
+							{ ! isBlazePro && <p className="login__form-terms">{ socialToS }</p> }
+							{ isWoo && ! isPartnerSignup && ! isWooPasswordless && this.renderLostPasswordLink() }
+							<div className="login__form-action">
+								<FormsButton
+									primary
+									busy={ ! isWoo && isSendingEmail }
+									disabled={ isSubmitButtonDisabled }
+								>
+									{ isWoo && isSendingEmail ? <Spinner /> : this.getLoginButtonText() }
+								</FormsButton>
+							</div>
 
-					{ ! hideSignupLink && isOauthLogin && (
-						<div className={ clsx( 'login__form-signup-link' ) }>
-							{ this.props.translate(
-								'Not on WordPress.com? {{signupLink}}Create an Account{{/signupLink}}.',
-								{
-									components: {
-										signupLink: <a href={ signupUrl } />,
-									},
-								}
+							{ ! hideSignupLink && isOauthLogin && (
+								<div className={ clsx( 'login__form-signup-link' ) }>
+									{ this.props.translate(
+										'Not on WordPress.com? {{signupLink}}Create an Account{{/signupLink}}.',
+										{
+											components: {
+												signupLink: <a href={ signupUrl } />,
+											},
+										}
+									) }
+								</div>
 							) }
-						</div>
+						</>
 					) }
 				</Card>
 
 				{ shouldShowSocialLoginForm && (
 					<Fragment>
 						<FormDivider />
 						<SocialLoginForm
-							linkingSocialService={
-								this.props.socialAccountIsLinking ? this.props.socialAccountLinkService : null
-							}
-							onSuccess={ this.props.onSuccess }
-							socialService={ this.props.socialService }
+							lastUsedAuthenticationMethod={ this.state.lastUsedAuthenticationMethod }
+							handleLogin={ this.handleSocialLogin }
+							trackLoginAndRememberRedirect={ this.trackLoginAndRememberRedirect }
+							reset={ this.resetLastUsedAuthenticationMethod }
```

Comment:
Updating this prop should fix the "Continue with Email" button:

```suggestion
							resetLastUsedAuthenticationMethod={ this.resetLastUsedAuthenticationMethod }
```